app/views/welcome/index.html.erb

```diff
@@ -1,4 +1,13 @@
-<div id="overlay"></div>    
+
+<div id="overlay">
+  <h1 id="overlay-title" > Having trouble with your heat? </h1>
+  
+  <h3 id="overlay-description"> At Heat Seek, our mission is to help people keep their heat on. If your building is without heat frequently, we may be able to help! We're seeking buildings to participate in our winter 2015 pilot program. Ideal buildings (1) have an organized tenant association, (2) have access to legal representation, and (3) are considering bringing a case to housing court this winter. If that sounds like your building, please reach out! </h3>
+  <div id="testimonials" ><span id="testimonial-span" class="hvr-float-shadow"> Read our testimonials </span> </div>
```

Comment:
This should say "Don't forget to contact 311" and link to here: https://www1.nyc.gov/nyc311-hpd-service-request/form.htm?execution=e2s1